### PR TITLE
Review Network Policy Ingress rules during Service debugging

### PR DESCRIPTION
**Pull request by chirangaalwis**
**Description:**
Add missing note to review [Kubernetes Network Policy Ingress rules](https://kubernetes.io/docs/concepts/services-networking/network-policies/) during debugging Kubernetes Service issues.

### CODE REVIEW

File: content/en/docs/tasks/debug-application-cluster/debug-service.md [hunk 1]
```diff
@@ -213,6 +213,14 @@ In order to highlight the full range of configuration, the Service you created
 here uses a different port number than the Pods.  For many real-world
 Services, these values might be the same.
 
+## Any Network Policy Ingress rules affecting the target Pods?
+
+If you have deployed any Network Policy Ingress rules which may affect incoming
+traffic to `hostnames-*` Pods, these need to be reviewed.
+
+Please refer to [Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+for advanced details.
```
Comment (tengqm) [inline content/en/docs/tasks/debug-application-cluster/debug-service.md:222]:
```suggestion
Please refer to [Network Policies](/docs/concepts/services-networking/network-policies/) for more details.
```

Comment (AvineshTripathi) [reply content/en/docs/tasks/debug-application-cluster/debug-service.md:222]: Hey @tengqm can you tell why we follow the url in a format `/docs/concept` and not `https://` as I have seen the Url format you suggested would give error in github while the other url won't give in github as well as actual website

Comment (chirangaalwis) [reply content/en/docs/tasks/debug-application-cluster/debug-service.md:222]: Oh my bad @tengqm, missed this. +1 adding.

**Comment by chirangaalwis** [comment]
@tengqm updated. Please re-review.

File: content/en/docs/tasks/debug-application-cluster/debug-service.md (hunk 1 above)
Comment (geoffcline) [reply content/en/docs/tasks/debug-application-cluster/debug-service.md]: @AvineshTripathi 

if you use the absolute or full url (`https://kubernetes.io/docs/concept`), the link will always direct the browser to that path at *the latest main version*.

if you use the relative form (`/docs/concept`), the link will direct the browser to that path at the *version the user was previously browsing*. such as https://v1-20.docs.kubernetes.io/

**Comment by divya-mohan0209** [comment]
@chirangaalwis : Please squash & merge commits into one. Lgtm to me otherwise. Shall approve once done.

**Comment by chirangaalwis** [comment]
> @chirangaalwis : Please squash & merge commits into one. Lgtm to me otherwise. Shall approve once done.

@divya-mohan0209 @geoffcline I have squashed the commits into one as requested. Please review and merge.

**Pull request merged**